### PR TITLE
[Release.1.3.2-rc2] [AzureRT]

### PR DESCRIPTION
**Pull request by hyonholee**
- Add ApplicationGatewayBackendAddressPoolsId parameter to New-AzureRmVmssIpConfig cmdlets
- Fix localization issue of New-AzureRmVM
- Remove InvokeAzureComputeMethod, New-AzureComputeArgumentList, New-AzureComputeParameterObject cmdlets

### CODE REVIEW

File: src/ResourceManager/Compute/Commands.Compute/Generated/InvokeAzureComputeMethodCmdlet.cs [hunk 1]
```diff
@@ -30,7 +30,7 @@
 
 namespace Microsoft.Azure.Commands.Compute.Automation
 {
-    [Cmdlet(VerbsLifecycle.Invoke, "AzureComputeMethod", DefaultParameterSetName = "InvokeByDynamicParameters")]
+    //[Cmdlet(VerbsLifecycle.Invoke, "AzureComputeMethod", DefaultParameterSetName = "InvokeByDynamicParameters")]
```
Comment (hovsepm) [inline src/ResourceManager/Compute/Commands.Compute/Generated/InvokeAzureComputeMethodCmdlet.cs]: @hyonholee why do you have commented code here?

Comment (hyonholee) [inline src/ResourceManager/Compute/Commands.Compute/Generated/InvokeAzureComputeMethodCmdlet.cs]: This is a generated file.  We decided not to generate Invoke-AzureComputeMethod cmdlet, which is a general cmdlet for invoking any compute APIs.  We modified our auto-generator such that it has an option to choose if it generates the cmdlet.  If we don't want, the generator just comments out.

**Comment by hovsepm** [comment]
on-demand: http://azuresdkci.cloudapp.net/view/1-AzurePowerShell/job/powershell-demand/682/

**Comment by hyonholee** [comment]
Tests for the fix is done manually and automated tests will be added later.

**Pull request merged**